api_proxy.yaml

```diff
@@ -488,13 +488,11 @@ definitions:
     required:
       - platform
       - pushChannel
-  ProfileWithEmail:
+  InitializedProfile:
```

Comment:
can't this be a reference to the ExtendedProfile model from APIs ?

Reply:
not yet, since it's a mix of fields coming from SPID and the API, the next step is to make that change, see PR #322